test/requestability_determination.test.js

```diff
@@ -0,0 +1,41 @@
+var requestableBasedOnStatusAndHoldingLocation = require('../lib/requestability_determination').requestableBasedOnStatusAndHoldingLocation
+
```

Comment:
Test the corner case of SCSB saying something is not available but our serialized status saying it is.